File: LeetCode/1662.Check_If_Two_String_Arrays/main.py
#!/usr/bin/python

from solution import Solution

# Unit Test
word1 = ["ab", "c"]
word2 = ["a", "bc"]

word11 = ["a", "cb"]
word22 = ["ab", "c"]

s = Solution()
if (s.numIdenticalPairs(word1, word2) == True):
    print("Pass")
else:
    print("Test 1 Failed")
if (s.numIdenticalPairs(word11, word22) == False):
    print("Pass")
else:
    print("Test 2 Failed")

File: LeetCode/1662.Check_If_Two_String_Arrays/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    vector<string> w1 {"ab", "c"};
    vector<string> w2 {"a", "bc"};
    vector<string> w11 {"a", "cb"};
    vector<string> w22 {"ab", "c"};
    if (s.arrayStringsAreEqual(w1, w2) == true) {cout << "Pass" << endl; } else {cout << "Failed 1" << endl; }
    if (s.arrayStringsAreEqual(w11, w22) == false) {cout << "Pass" << endl; } else {cout << "Failed 2" << endl; }
    
    return 0;
}